benchmark/cuchembm/metrics/sampling.py

```diff
@@ -161,9 +172,9 @@ def variations(self, cfg, **kwargs):
         radius_list = [float(x) for x in radius_list]
         return {'radius': radius_list}
 
-    def smiles_in_train(self, smiles):
+    def smiles_not_in_train(self, smiles):
         in_train = self.training_data.is_known_smiles(smiles)
-        return in_train
+        return not(in_train)
```

Comment:
@rilango Note: I believe there was a bug in this metric -- should be counting the number of molecules NOT in training data.